app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -271,6 +271,14 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
     }
   };
 
+  $scope.isDetectionEnabled = function() {
```

Comment:
Seems fine. I would not stop detection when we have all the data and it has not yet been validated.

There should be an error message if the detection fails.